manifests/server/conf.pp

```diff
@@ -161,10 +165,10 @@
     include '::haveged'
   }
 
+  $rhel_greater_then_6 = ( $facts['os']['name'] in ['RedHat','CentOS','Fedora'] ) and ( $facts['os']['release']['major'] > '6' )
```

Comment:
The logic in client/params.pp checks if the Fedora version is > 22.  So a couple of questions about this:
* Should the logic be the same
* Why are we referencing Fedora when the metadata.json does not list Fedora in list of supported OSs
* Why not use simplib::assert_metadata( $module_name )

Reply:
1. The logic should be the same, I can fix that
2. I don't mind Fedora in the logic, but we can't test it in this ticket. Would removing it be an issue?